librz/type/function.c

```diff
@@ -154,6 +153,17 @@ RZ_API bool rz_type_func_save(RzTypeDB *typedb, RZ_NONNULL RzCallable *callable)
 	return true;
 }
 
+/**
+ * \brief Update RzCallable type in the types database
+ *
+ * \param typedb Type Database instance
+ * \param callable RzCallable type to save
+ */
+RZ_API bool rz_type_func_update(RzTypeDB *typedb, RZ_NONNULL RzCallable *callable) {
+	rz_return_val_if_fail(typedb && callable && callable->name, false);
+	return ht_pp_update(typedb->callables, callable->name, callable);
```

Comment:
Should it free the old one first? To avoid the leak?

Reply:
Yes, I forgot it.

Reply:
I checked and found that `reserve_kv` will have free old one.

Reply:
Perfect, no problem then.